### PR TITLE
Add admin tag manager documentation to edit and delete tags

### PR DESCRIPTION
**Pull request by amajai**
Solution to #1220 .
Added admin tag manager solution to the documentation where you can set up admin tag interface to manage tags.

### CODE REVIEW

File: docs/reference/pages/model_recipes.rst [hunk 1]
```diff
@@ -255,3 +255,33 @@ By default, tag fields work on a "free tagging" basis: editors can enter anythin
             verbose_name_plural = "blog tags"
 
 Here we have registered ``BlogTag`` as a snippet, to provide an interface for administrators (and other users with the appropriate permissions) to manage the allowed set of tags. With the ``free_tagging = False`` option set, editors can no longer enter arbitrary text into the tag field, and must instead select existing tags from the autocomplete dropdown.
+
+Admin tag manager
+----------------------
```
Comment (lb-) [inline docs/reference/pages/model_recipes.rst:260]: Maybe something a bit clearer might be better...

```
Managing tags with Wagtail's `ModelAdmin`
-----------------------------------------
```

**Review by lb- — CHANGES_REQUESTED** [review]
Looking good, just a minor nitpick on the heading.

**Review by amajai — COMMENTED** [review]
Thanks. Much clearer on describing the model recipe. Will change it right away.

**Review by lb- — APPROVED** [review]
looking great @amajai  - I have rested the code as documented locally and works really great!

I have pushed to your remote branch with a few tweaks in formatting and also the changelog/release notes entry.

I have added you as a contributor as `Abdulmajeed Isa` - please let me know if you want your name listed differently.

Thanks so much for taking the time to contribute to Wagtail!

**Pull request merged**